### DISCUSSION TITLE
How to get timelapse export to work correctly?

### DISCUSSION THREAD

**Discussion by dolai1** [blakeblackshear/frigate#14075] [Ask A Question]
Hello... I am still running 0.13.2. 

When use Exports and select a camera, "Realtime", and a time interval, a file is generated and it plays back normally and as expected.

When I instead select "Timelapse", the output video, however, has one new frame about every second, though VLC advises that the framerate is 30/s. The total duration of the file is about right for a timelapse.

I have tried several custom timelapse_args lines in the config, record>export>timelapse_args, without a change in the undesired behavior.

How do I get an actual framerate of 30/s as desired? Thanks.

**Comment by dolai1** [comment]
This looks like it's related to the codec used by the camera [Dahua (EmpireTech) IPC-T58IR-ZE-S3 with H.264H "Smart Codec" VBR, in this case] as that from other models seems to work properly.

So, I suppose that the question changes... what do I need to do to the "timelapse_args" line to make it work with this camera's output?

Changing the codec is not an option in this case, as the video has already been recorded, and this takes up /much/ less storage space than alternatives.

**Reply by NickM-27** [reply]
Frigate does not recommend using proprietary protocols, you could re-encode and then try to re-encode again as a timelapse but there is no telling what changes they are making here.

**Reply by dolai1** [reply]
I am downloading the realtime and reprocessing manually; thanks.

It looks like the timelapse feature was deprecated in 0.14; here's one vote to keep it in... and have it capable of full processing... I use it to document work that I am doing on the property. It's great to be able to see large/slow changes over a short period of time.

Thank you for the work that you do.

**Comment by LukeHandle** [comment]
> the output video, however, has one new frame about every second, though VLC advises that the framerate is 30/s. The total duration of the file is about right for a timelapse.

I believe this is similar to what I found in https://github.com/blakeblackshear/frigate/discussions/14126 - there are some options used that         speed up the export of the video by seemingly skipping through the input video (vs. reading every frame). As a workaround, you could do a normal export, then exec into the container and run a manual timelapse command like I showed there.

**Reply by dolai1** [reply]
Ah, yeah.
So I guess it's taking shortcut(s). And, it looks like the timelapse feature was deprecated, unfortunately.
Like you, I want to be able to show work on the property.
So, manual reprocessing for now and more than a day is out of the question...
Thank you.
